recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
   run:
     - python
     - conda-build-all
-    - conda
+    - conda 4.1.*
```

Comment:
This is temporary until we can figure out how to make `conda-smithy` work with `conda` version `4.2.x`. Unfortunately, we know `conda` will likely squirm out of this constraint at present. So we should keep an eye on it.